pkg/resourcemapping/v2/orm_client_v2.go

```diff
@@ -41,8 +42,10 @@ type ORMv2Client struct {
 	*registry.ResourceMappingRegistry
 }
 
+// TODO: Passing the event recorder as nil as of now, will have to come up with creating
+// own event recorder during this initialization of toolbox to capture events
 func NewORMv2Client(kubeconfig *restclient.Config) (*ORMv2Client, error) {
-	err := kubernetes.InitToolbox(kubeconfig, scheme)
+	err := kubernetes.InitToolbox(kubeconfig, scheme, nil)
```

Comment:
Please log a task to work on incorporating the event recorder for recording action events

Reply:
I did log a separate task for this - https://jsw.ibm.com/browse/TRB-40046